src/components/registration/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const cards = () => {
+  return (
+    <div className="flex justify-center w-full md:hidden ">
+      <div className="max-w-sm flex items-center h-screen overflow-hidden w-full">
+        <div className="flex justify-center items-center h-screen flex-col w-full">
+          <div className="">
+            <button
+              type="submit"
```

Comment:
`type` should be button and not `submit`, there is no form at the top and make a dummy link of where this should link to. Can we do the same thing for the get started button to?